dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityLiveMode.java

```diff
@@ -74,7 +61,7 @@ public final void serve(final OutputStream out) throws DotDataException, IOExcep
 
 
             // now we check identifier cache first (which DOES NOT have a 404 cache )
-            Identifier id = APILocator.getIdentifierAPI().find(host, uri);
+            Identifier id = APILocator.getIdentifierAPI().find(this.htmlPage.getIdentifier());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'id' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4748140)